### DISCUSSION TITLE
Zora Network - Verifying Contracts

### DISCUSSION THREAD

**Discussion by alexanderattar** [blockscout/blockscout#8257] [Q&A]
Hello,

I am trying to verify the @holographxyz protocol contracts on the Zora network, but cannot seem to find any option to login and get an API key. 

Is this not currently available?

**Comment by eferbarn** [comment]
Hi, 
I have the same problem. Please help me. I need an API key but I can't find a login option in Explorer!

**Reply by GimluCom** [reply]
Try `apiKey: "abc"`. The default behavior of Blockscout is to accept any arbitrary key.
